Probe/src/data_collector.py

```diff
@@ -0,0 +1,39 @@
+import scapy.all as scapy
+import os
+import datetime
+
+
+def dataCapture(noOfPackets = 2,filename = "f"):
```

Comment:
I don't see any exception handling, handle exceptions in every possible control flow